TGC/localisation/00_events_1000000000.csv

```diff
@@ -1,15 +1,15 @@
 CODE;ENGLISH;FRENCH;GERMAN;POLISH;SPANISH;ITALIAN;SWEDISH;CZECH;HUNGARIAN;DUTCH;PORTUGUESE;RUSSIAN;FINNISH;x
-EVTDESC1904191417;The call has come for Tabasco to join Yucat�n, a union of two brother-states which will tend to the needs of their people.;;;;;;;;;;;;;x
```

Comment:
Your editor was set to UTF-8 (probably) instead of Windows-1252 charset and so all the non-ASCII characters will get fucked up. Restore this file and then reapply your changes saved as Windows-1252 charset.